rp/src/key.rs

```diff
@@ -0,0 +1,105 @@
+use std::{fs, path::Path};
+
+use anyhow::{anyhow, Result};
+use wireguard_keys::Privkey;
+
+use rosenpass::protocol::{SPk, SSk};
+use rosenpass_cipher_traits::Kem;
+use rosenpass_ciphers::kem::StaticKem;
+use rosenpass_secret_memory::file::StoreSecret as _;
+
+pub fn genkey(private_keys_dir: &Path) -> Result<()> {
+    if private_keys_dir.exists() {
```

Comment:
I think it is okay if this directory exists. It should just not overwrite existing files.

Reply:
Resolved in ad10f3bcc6b684a4fd9338794d1e94e1f7cac908.